background/KF.ts

```diff
@@ -601,17 +602,17 @@ class KeeFox {
     RegularKPRPCListenerQueueHandler ()
     {
         // If there is nothing in the queue at the moment or we are already processing a callback, we give up for now
-        if (keefox_org.processingCallback || keefox_org.pendingCallback == "")
+        if (this.processingCallback || this.pendingCallback == "")
```

Comment:
I find the use of `keefox_org` weird (wrong)
Inside of `Keefox` declare the class variable `processingCallback`
you instantiate the class and use the variable of the referenced class.

```
class KeeFox {
  processingCallback: boolean;

  KPRPCListener(sig){
  keefox_org.processingCallback = true;
  }
}

keefox_org = new KeeFox();

```
an instance of the class itself should not be used inside of itself.